src/main/tv/codely/scala_intro_examples/lesson_08_collections/WordCounter.scala

```diff
@@ -1,9 +1,10 @@
 package tv.codely.scala_intro_examples.lesson_08_collections
 
 final class WordCounter {
-  // @ToDo: Here you have your exercise!
-  // @see tv.codely.scala_intro_examples.lesson_08_collections.WordCounterSpec
   def countWords(text: String): Int = {
-    0
+    if (text.isEmpty)
+      return 0
```

Comment:
`return` statement not needed. In fact, it's not Scala idiomatic to specify such clauses since we will always return the last statement in a code block. In order to remove it, we should add the `else` part of the `if` 🙂